dataflow/src/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -478,7 +480,9 @@ public boolean equals(Object obj) {
                 return false;
             }
             LocalVariable other = (LocalVariable) obj;
-            return other.element.equals(element);
+            return other.element.name.toString().equals(element.name.toString()) &&
+                   other.element.type.toString().equals(element.type.toString()) &&
```

Comment:
I'm adding a comment that describes the reason I believe this is not used - please have a look once I posted the new diff.